cms-adapter/src/Rsbc.Dmf.CaseManagement.Service/Services/CaseService.cs

```diff
@@ -853,12 +853,22 @@ public async override Task<ResultStatusReply> CreateBringForward(BringForwardReq
         {
             ResultStatusReply reply = new ResultStatusReply();
 
-            // fetch the document.
             try
             {
+                var bringForwardRequest = new CaseManagement.BringForwardRequest()
+                {
+                    CaseId = request.CaseId ?? string.Empty,
+                    Assignee = request.Assignee ?? string.Empty,
+                    Description = request.Description?? string.Empty,                
+                    Subject = "ICBC Error",
```

Comment:
The subject should come from the BringForwardRequest rather than always being ICBC Error.

Reply:
Fixed the issue with Subject to get from BringForwardRequest